src/main/java/com/agency04/sbss/pizza/Francos.java

```diff
@@ -0,0 +1,20 @@
+package com.agency04.sbss.pizza;
+
+import org.springframework.stereotype.Component;
+
+public class Francos implements PizzeriaService{
+    @Override
+    public String getName() {
+        return null;
+    }
+
+    @Override
+    public String getAddress() {
+        return null;
+    }
+
+    @Override
+    public Pizza makePizza(Pizza pizza) {
+        return pizza;
+    }
```

Comment:
Ovu pizzeriu bi tribalo anotirat sa `@Component` jer zelis imat ovaj serice u spring contextu. Uz to, `getName` i ` getAddress` metode bi tribale vraćat nesto smisleno, a ne `null`. U ovom slucaju bi bilo pametnije metode skroz izbrisat nego ih imat ovakve. Mozda bi moga prominit te metode da vraćaju nesto smisleno pa ih iskoristit u `makePizza` metodi, npr. ovako nekako:
```
   @Override
    public String makePizza(Pizza pizza) {
        return "getName() + " na adresi: " + getAddress() + "priprema pizzu: " + pizza;
    }
```

Ista stvar i za drugu pizzeriu